### ISSUE TITLE
Component updated every 16ms.

### ISSUE THREAD

**Issue by ncnlinh** [jaysoo/react-native-menu#9]
setState in `MenuContext`, which is called using controller when Menu is updated refresh the state again, which ends in a infinite loop of setState.

**Comment by ncnlinh** [comment]
```
// Options haven't changed, no need to re-register.
if (this._options[name] === options) {
        return;
}
```

The function will always register since `options` are objects and the condition will always return false.

**Comment by jaysoo** [comment]
@ncnlinh  Thanks for reporting. Just published `0.19.0 that solves this issue.

**Comment by 24ark** [comment]
I am facing the same issue after updating to 0.20.0 .
